lib/web_ui/lib/src/engine/bitmap_canvas.dart

```diff
@@ -714,6 +714,10 @@ class BitmapCanvas extends EngineCanvas {
       rootElement.append(paragraphElement);
     }
     _children.add(paragraphElement);
+    // If there is a prior sibling such as img prevent left/top shift.
```

Comment:
Is there any information about why this is happening? Would be useful to leave a link explaining what causes the shift.

Reply:
No info or chromium bug i could point to. Although position is absolute and the paragraph has transform on it, it is shifting the text vertically down.

Reply:
/me pulling my hair out 🤦